db/seeds.sql

```diff
@@ -1,2 +1,36 @@
--- INSERT INTO users (username, first_name, last_name, street_address1, city, _state, zip, email_address, phone1) 
--- VALUES ('jaxdot', 'jacqueline', 'gaherity', '15771 oxford way', 'sandwich', 'ma', '98012', 'jacquelne.gaherity@UW.edu', '508-888-9755');
+/* */
+
+use project2_db;
+-- drop database if exists project2_db;
+   create database if not exists project2_db;
+-- DROP TABLE IF EXISTS inventory;
+-- DROP TABLE  IF EXISTS users;
+-- DROP TABLE IF EXISTS logins
+-- DROP TABLE IF EXISTS purchase_orders;
+
+INSERT INTO users (username, first_name, last_name, street_address1, city, _state, zip, email_address, phone1) 
```

Comment:
This is fine, but you might want to look up multi insert to save time: http://www.mysqltutorial.org/mysql-insert-statement.aspx

Reply:
Yeah, that looks good for next time.